assets/styling.css

```diff
@@ -1,233 +1,4 @@
-        url('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted.eot?#iefix') format('embedded-opentype'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted.woff') format('woff'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted.ttf') format('truetype');
-    font-weight: bold;
-    font-style: italic;
-}
-
-@font-face {
-    font-family: 'Brown';
-    src: url('https://www.katacoda.com/daml/assets/Brown-BoldItalic.eot');
-    src: local('https://www.katacoda.com/daml/assets/Brown-BoldItalic'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldItalic.eot?#iefix') format('embedded-opentype'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldItalic.woff') format('woff'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldItalic.ttf') format('truetype');
-    font-weight: bold;
-    font-style: italic;
-}
-
-@font-face {
-    font-family: 'Brown';
-    src: url('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted-Alt.eot');
-    src: local('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted-Alt'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted-Alt.eot?#iefix') format('embedded-opentype'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted-Alt.woff') format('woff'),
-        url('https://www.katacoda.com/daml/assets/Brown-BoldBackslanted-Alt.ttf') format('truetype');
-    font-weight: bold;
-    font-style: italic;
-}
-
-/*-------------------Btn Font------------------------*/
-@font-face {
-    font-family: 'NB International';
-    src: url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont.woff2') format('woff2'),
-        url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont.woff') format('woff'),
-        url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont.ttf') format('truetype');
-    font-weight: normal;
-    font-style: normal;
-    font-display: swap;
-}
-
-@font-face {
-    font-family: 'NB International';
-    src: url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont_1.woff2') format('woff2'),
-        url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont_1.woff') format('woff'),
-        url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont_1.ttf') format('truetype');
-    font-weight: normal;
-    font-style: normal;
-    font-display: swap;
-}
-
-@font-face {
-    font-family: 'NB International';
-    src: url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont_2.woff2') format('woff2'),
-        url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont_2.woff') format('woff'),
-        url('https://www.katacoda.com/daml/assets/NBInternationalMonoWebfont_2.ttf') format('truetype');
-    font-weight: normal;
-    font-style: normal;
-    font-display: swap;
-}
-/* --- FONTS END ---- */
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
```

Comment:
Shouldn't there be an equivalent line for Roboto Mono?

Reply:
Worked without it as well, but I've added the import line just to be safe

Reply:
Maybe because you have them cached or installed on your machine. Thanks.